apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -109,9 +109,7 @@ export function calendarizeFinals(currentCourses: ScheduleCourse[] = []): Course
              * Fallback to January 2018 if no finals start date is available.
              * defaultFinalsDay is handled later by day since it varies by day.
```

Comment:
Just a small thing: change `defaultFinalsDay` to `finalsDay` in this comment since we renamed the variables